Makefile

```diff
@@ -82,6 +82,10 @@ blobfuse-windows:
 	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/blobfuseplugin.exe ./pkg/blobfuseplugin
 
+.PHONY: container
```

Comment:
`blobfuse-container` has existed which executes the same command.

Reply:
yes, I would to create a new generic command as `make container`, it will be applied to all csi drivers, while original `make blobfuse-container` should be also there for compatability.